moped-editor/src/views/projects/projectView/ProjectNameComplex.js

```diff
@@ -0,0 +1,292 @@
+import React, { useState } from "react";
+import makeStyles from "@mui/styles/makeStyles";
+import { Grid, Icon, Box, Typography, TextField, Snackbar, Alert } from "@mui/material";
+import ProjectStatusBadge from "./ProjectStatusBadge";
+import { useMutation } from "@apollo/client";
+import { UPDATE_PROJECT_NAMES_QUERY } from "../../../queries/project";
+
+const useStyles = makeStyles((theme) => ({
+  editIcons: {
+    cursor: "pointer",
+    marginTop: "16px",
+    fontSize: "2rem",
+  },
+  projectNameEditField: {
+    "font-size": "1.5rem",
+    "font-weight": "bold",
+  },
+  projectSecondaryNameEditField: {
+    "font-size": "1.0rem",
+    "font-weight": "bold",
+    "padding-top": "0.95rem",
+  },
+}));
+
+const ProjectNameComplex = (props) => {
+  // Incoming props:
+    // projectData={data.moped_project[0]} ✅
+    // projectId={projectId} ✅
+    // editable={true} ❓
```

Comment:
I never really found this being used in our previous `ProjectNameEditable` component, but it's passed in as a prop. 🤷

Reply:
this feels like it must be a holdover from once upon a time when the project name was not editable inline there. i think it can be removed